py/picca/io.py

```diff
@@ -510,8 +510,11 @@ def read_from_spplate(in_dir, thid, ra, dec, zqso, plate, mjd, fid, order, log=N
         mjd_spall = spAll[1]["MJD"][:]
         fid_spall = spAll[1]["FIBERID"][:]
         qual_spall = spAll[1]["PLATEQUALITY"][:]
+        zwarn_spall = spAll[1]["ZWARNING"][:]
 
         w = sp.in1d(thid_spall, thid) & (qual_spall == b"good")
+        for zwarnbit in [0,1,7,8,9]:
```

Comment:
@londumas could you add a comment of what you are rejecting and a link to the zwarn maskbits?